### ISSUE TITLE
[Content Author Module 4: Images] [medium[ Complex image alternative data visualization

### ISSUE THREAD

**Issue by iadawn** [w3c/wai-curricula#605]
In the learning outcomes for Complex Images there is the following outcome:

> provide alternative data visualizations for complex images, including text descriptions, data tables, and other graphical representations

I am not sure what this means. An 'alternative data visualization' for a complex image would be another complex image if it is a visualization.

**Comment by daniel-montalvo** [comment]
Hey @iadawn 

I am assuming this comment implies that data visualization should be restricted to content perceived uniquely through vision. So data tables and long descriptions for complex images won't be considered "data visualizations" but instead "alternatives to data visualizations".

in [Topic Complex Images](https://content-author-modules--wai-curricula.netlify.app/curricula/content-author-modules/images/#topic-complex-images), I have changed the learning outcome to:

> provide other graphical representations as alternative data visualizations for complex images

In addition, in module [Data Tables](https://content-author-modules--wai-curricula.netlify.app/curricula/content-author-modules/data-tables/), I have made the following changes.

* Learning outcomes for module, second bullet: 
  > provide data tables as an alternative to data visualizations contained in charts, diagrams, and infographics
* Ideas to assess knowledge for module, first bullet: 
  > Practical &mdash; Give students a chart, diagram, or infographic and ask them to provide an alternative to that data visualization using a regular data table. Assess how students provide alternatives to data visualizations contained in charts, diagrams, and infographics using regular data tables.
  
Is this what you meant?
